content/en/platform/corda/5.0/application-networks/creating/mgm/register.md

```diff
@@ -152,4 +153,20 @@ Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -
 ```
 {{% /tab %}}
 {{< /tabs >}}
-If successful, you should see the `APPROVED` registration status.
+
+The returned status of the registration request will be one of the following options:
+
+* `APPROVED`: The MGM was onboarded successfully.
```

Comment:
```suggestion
* `APPROVED`: The member was onboarded successfully.
```

Reply:
Well, technically speaking the MGM can register with itself and it would also see APPROVED to indicated the MGM onboarded successfully, but typically it would be members registering so the APPROVED status would indicate a member was onboarded. More broadly the description here could be "The registration request passed all validation and was approved by the MGM"

Reply:
Updated the definition.